README.md

```diff
@@ -76,86 +81,109 @@ Once you have done so, install Ruby 2.1+ (probably 2.3.0 at time of writing). Th
+The syntax of the letsencrypt_webfaction command is as follows:
 
-[The changelog](./CHANGELOG.md) describes changes from version to version.
+    $ letsencrypt_webfaction --account_email <email-address> --domains <domain[,domain[,domain...]]> --public <server-folder>
 
-LetsEncrypt Webfaction follows [Semantic Versioning](http://semver.org/). In a nutshell, a version number such as `1.2.3` is divided as `major.minor.patch`. When the major version is incremented, you will probably have to change something about the configuration to make it work. The changelog will let you know what changes you need to make. When the minor version is incremented, there are new features but existing features haven't changed. If the patch version is incremented, the changes are all under the hood and shouldn't change or add any existing features.
+    
 
-TL;DR: Be careful with major version upgrades and you should be fine with upgrading to minor or patch releases.
+### Options:
 
-To upgrade, run one of the following two commands to fetch and install the newest version from RubyGems:
+    --account_email
 
-```sh
-# For system Ruby:
-GEM_HOME=$HOME/.letsencrypt_webfaction/gems RUBYLIB=$GEM_HOME/lib gem2.2 install letsencrypt_webfaction
+        Your WebFaction contact email address.
 
-# For RBenv
-RBENV_VERSION=2.3.0 gem install letsencrypt_webfaction
-```
+    --domains
+        
+        The domains for which you want to create certificates, separated by commas (with no spaces). The domains must be served from the same folder. There is one certificate per webapp, regardless of how many domains are served from it.
 
-## Usage
+    --public
 
-Here is a basic example which issues one certificate for both example.com and www.example.com which is served by `~/webapps/myapp/my_public_html` when your WebFaction contact email address is myemail@example.com. This assumes that both example.com and www.example.com are served from the same folder. An easy way to think about it is that there is one certificate per webapp, regardless of how many domains are served from it.
+        A folder which is directly served at `http://[yourdomain.com]/` into which the ACME verification files can be placed.
 
-    $ letsencrypt_webfaction --account_email [myemail@example.com] --domains [example.com,www.example.com] --public ~/webapps/[myapp]/[my_public_html]/
+        In the case of a PHP site, such as Drupal or Wordpress, look for the folder with `index.php` in it. This is usually in `/home/[myuser]/webapps/[yourapp/]`.
 
-The certificate will be placed in `~/le_certs/example.com/[timestamp]/` and WebFaction will be emailed to install the certificate.
+        In the case of a Rails app, look for a folder called `public/`. If you are deploying your app with Capistrano, this could show up in `/home/myuser/webapps/[yourapp]/current/public/`. 
```

Comment:
Markdown views indents as code examples. You can see in the [full rendered file](https://github.com/ElectricFeet/letsencrypt-webfaction/blob/ce384ffe7e291ae1637aa663e788c0681188db50/README.md) that you have to scroll left/right to read it all. I'd suggest using backticks (`) for the argument and then normal indentation for the description.